README.md

```diff
@@ -94,12 +51,68 @@ This will inject `APP_KEY` and `DATABASE_URL` using your service name and stage,
 
 Finally, deploy your app:
 
+```bash
+serverless deploy
```

Comment:
I went with the simplest command by default (simpler for new users, who don't have to wonder why they have to provide a stage)